umbra-js/src/classes/Umbra.ts

```diff
@@ -26,7 +26,7 @@ import { RandomNumber } from './RandomNumber';
 import { blockedStealthAddresses, lookupRecipient } from '../utils/utils';
 import { Umbra as UmbraContract, Erc20 as ERC20 } from '@umbra/contracts/typechain';
 import { ERC20_ABI } from '../utils/constants';
-import type { Announcement, ChainConfig, EthersProvider, ScanOverrides, SendOverrides, SubgraphAnnouncement, UserAnnouncement } from '../types'; // prettier-ignore
+import type { Announcement, ChainConfig, EthersProvider, ScanOverrides, SendOverrides, SubgraphAnnouncement, UserAnnouncement, AnnouncementDetail } from '../types'; // prettier-ignore
```

Comment:
(arbitrary comment location)

scanning seems to make no progress when the Umbra tab doesn't have focus, so I can't switch tabs and let it scan in the background—any idea what's going on here?

Reply:
Good catch! I did not realize that. Let me do some investigation. If it's not a quick fix I'll make an issue.

Reply:
huh, that's annoying, but sounds good thanks!